pkg/locodedb/calls.go

```diff
@@ -24,7 +24,14 @@ func Get(locodeStr string) (Record, error) {
 		return Record{}, err
 	}
 
-	key, err := NewKey(locodeStr[:2], locodeStr[2:])
+	country := locodeStr[:2]
+	location := locodeStr[2:]
+
+	if len(locodeStr) == 6 {
```

Comment:
can we check that `[2]` is a real space char? otherwise, it can lead to strange results if i pass `RUMOW `

Reply:
updated